.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: "daily"
```

Comment:
In the other repositories we do it weekly. Did you chose daily on purpose ?

Reply:
Yes, I think that weekly is more relevant for repo's that are spammed by notifications, but if you think weekly is better I can change it.

Reply:
I don't know if they are a lot of pckages to update here. You know best :)